makefile

```diff
@@ -378,6 +378,9 @@ phony += clean
 gqlgen:
 	go run github.com/99designs/gqlgen generate
 
+swaggo: 
+	make swaggo-build && make swaggo-render
```

Comment:
The more make-esque way to do this is to just have the `swaggo` recipe depend on `swaggo-build` and `swaggo-render`, and then [PHONY it](https://stackoverflow.com/a/2145605):
```make
swaggo: swaggo-build swaggo-render
phony += swaggo swaggo-install swaggo-format swaggo-build swaggo-render

# Later on...
.PHONY: $(phony)
```

If possible, also putting the output results in the output `bin` directory would also be helpful to avoid clutter. If these targets are made right now, you would have to manually `rm` or `git clean` to clean them up.

Reply:
> The more make-esque way to do this is to just have the `swaggo` recipe depend on `swaggo-build` and `swaggo-render`, and then [PHONY it](https://stackoverflow.com/a/2145605):

I thought I couldn't do this because I can't depend on them executing serially and in the order defined.

> If possible, also putting the output results in the output `bin` directory would also be helpful to avoid clutter. If these targets are made right now, you would have to manually `rm` or `git clean` to clean them up.

Done.

Reply:
Seems like the make spec doesn't guarantee execution order, so that's fine. Can we still add them to the phony list since they're not recipes for real files?